apps/molgenis-components/src/components/forms/InputRefList.vue

```diff
@@ -9,12 +9,15 @@
             :label="flattenObject(item)"
             @click="deselect(key)"
           />
-          <div>
-            <ButtonAlt class="pl-1" icon="fa fa-clear" @click="clearValue">
-              clear selection
-            </ButtonAlt>
-          </div>
         </div>
+        <ButtonAlt
+          v-if="value && value.length"
+          class="pl-1"
+          icon="fa fa-clear"
```

Comment:
I don't see the "fa-clear" icon (chrome, ff and safari) is the icon there for you?